openg2p_registration/models/openg2p_create_registration.py

```diff
@@ -0,0 +1,319 @@
+from odoo import api, fields, models, SUPERUSER_ID
+from odoo.exceptions import UserError, ValidationError, _logger
+from odoo.tools.translate import _
+import time
+
+AVAILABLE_PRIORITIES = [("0", "Urgent"), ("1", "High"), ("2", "Normal"), ("3", "Low")]
+
+
+class CreateRegistration(models.Model):
+    _inherit = ["openg2p.registration"]
+
+    def create_registration_from_odk(self, odk_data):
```

Comment:
That's not how functions should be defined. Usw function overriding.

Reply:
Function names should not be defined for function parameters or where the function is being called from.